src/components/Game.java

```diff
@@ -1,21 +1,86 @@
 package components;
 
 import lib.Node;
+import ui.Main;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 
 public class Game extends Node {
     public String title;
     public String description;
     public int lives;
-    public Scene start;
-    public List<Integer> stats = new ArrayList<>();
+    public Scene startScene;
+    public static HashMap<String, Integer> numberStats = new HashMap<>();
+    public static HashMap<String, String> stringStats = new HashMap<>();
+    public List<Node> storyScenes = new ArrayList<>();
+    public List<Node> deathScenes = new ArrayList<>();
+    public List<Node> endScenes = new ArrayList<>();
 
 
     @Override
     public void parse() {
-
+        tokenizer.getAndCheckNext("GAME");
+        while (tokenizer.moreTokens()) {
+            String statKey;
+            String statValue;
+            Scene scene;
+            String currToken = tokenizer.getNext();
+            switch (currToken) {
+                case "title":
+                    title = tokenizer.getNext();
+                    break;
+                case "description":
+                    description = tokenizer.getNext();
+                    break;
+                case "lives":
+                    lives = Integer.parseInt(tokenizer.getNext());
+                    if (lives < 0) {
+                        System.out.println("Lives can't be less than 0");
+                        System.exit(1);
+                    }
+                    break;
+                case "stats":
+                    // adds next 2 tokens to stats if they aren't one of the literals
+                    while(!Main.literals.contains(tokenizer.checkNext())) {
+                        statKey = tokenizer.getNext();
+                        statValue = tokenizer.getNext();
+                        // tries parsing statValue as an int, and if it fails, then it places it in the stringStats
+                        try {
+                            numberStats.put(statKey, Integer.parseInt(statValue));
+                        } catch (NumberFormatException e) {
+                            stringStats.put(statKey, statValue);
```

Comment:
Is this going to be trimmed? I trimmed off the start/end double quotes. I'll leave it in if this won't be trimmed

Reply:
Are you talking about trimming white space here, or quotes? if white space, we're already doing that in the tokenizer

Reply:
oh, yeah i can probably trim off the quotes as well, i didnt do that here